docs/context/ref/SearchCriteria.md

```diff
@@ -0,0 +1,60 @@
+---
+id: SearchCriteria
+sidebar_label: SearchCriteria
+title: SearchCriteria
+hide_title: true
+---
+# `SearchCriteria`
+
+A context type representing a search criteria that can be used to filter and display messages from a chat.
```

Comment:
Given the generic name, type name and structure (an array of context objects), the description should also be generic (but can use chat as an example).
```suggestion
A context type that represents a simple search criterion, based on a list of other context objects, that can be used to search or filter content in an application (such as messages in a chat app).
```

Reply:
I'm tempted to also suggest tweaking the name to acknowledge the simplicity of the criterion (a bag of contexts) in case we want to introduce more complex search types in future (but haven't come to a conclusion on that yet - after all we might expand this one with additional fields instead...).

Reply:
I also think an explanation of how multiple contexts in the list should be interpreted (to encourage similar behaviour). Should they be combined with AND or OR? 

If variation in behaviour is expected or desirable an enumerated field should be added for that (e.g. combinationLogic: "AND") or the contexts field replaced with separate fields (e.g. `contexts: { and: [...], or: [...], not: [...]}`).

Reply:
@kriswest I like your idea about using `contexts: { and: [...], or: [...], not: [...]}`
It's very generic :)
I'll update the PR accordingly.